packages/react-sdk/src/components/Button/CompositeButton.tsx

```diff
@@ -52,7 +61,16 @@ export const CompositeButton = forwardRef<
             variant === 'secondary' && active,
         })}
       >
-        {children}
+        <button
+          className={clsx('str-video__composite-button__button')}
```

Comment:
```suggestion
          className={clsx('str-video__composite-button__button')}
          type="button"
```
I think it is a good practice to add `type="button"` to all `<button>` elements as they otherwise may mess-up possible forms.

Reply:
Good catch!